src/javascript/_common/base/__tests__/client_base.js

```diff
@@ -166,13 +168,24 @@ describe('ClientBase', () => {
         });
         it('returns as expected for multi account opening', () => {
             State.set(['response', 'authorize', 'authorize', 'upgradeable_landing_companies'], [ 'svg' ]);
+            State.set(['response', 'landing_company', 'landing_company', 'legal_allowed_currencies'], true);
             Client.set('landing_company_shortcode', 'svg');
             const upgrade_info = Client.getBasicUpgradeInfo();
             expect(upgrade_info.can_upgrade).to.eq(false);
             expect(upgrade_info.can_upgrade_to.length).to.eq(0);
             expect(upgrade_info.type).to.eq(undefined);
             expect(upgrade_info.can_open_multi).to.eq(true);
         });
+        it('returns as expected for multi account opening without legal currencies', () => {
+            State.set(['response', 'authorize', 'authorize', 'upgradeable_landing_companies'], [ 'svg' ]);
+            State.set(['response', 'landing_company', 'landing_company', 'legal_allowed_currencies'], false);
```

Comment:
I think we should set a real value for it, to mimic what API will send.

Reply:
In the function itself, it is checking if it exists or not, therefore a boolean value, I have tried using 'USD' as we discussed but the test expects boolean

Reply:
@matin-binary then it won't work in real life because API sends a value, right? that means the function is not written correctly

Reply:
the test and real life should match, and the function should work the same way.